src/renderers/webgl/WebGLState.js

```diff
@@ -88,10 +88,36 @@ function WebGLState( gl ) {
 
 			setReversed: function ( value ) {
 
+				if ( reversed !== value ) {
+
+					const ext = extensions.get( 'EXT_clip_control' );
+
+					if ( reversed ) {
+
+						ext.clipControlEXT( ext.LOWER_LEFT_EXT, ext.ZERO_TO_ONE_EXT );
+
+					} else {
+
+						ext.clipControlEXT( ext.LOWER_LEFT_EXT, ext.NEGATIVE_ONE_TO_ONE_EXT );
+
+					}
+
+					const oldDepth = currentDepthClear;
+					currentDepthClear = null;
+					this.setClear( oldDepth );
```

Comment:
@eXponenta  Additionally, maybe we should also do:

```js
const oldFunc = currentDepthFunc;
currentDepthFunc = null;
this.setFunc( oldFunc );
```

Reply:
Oops. Yep, this one. 
Because we also should upgrade depth func to inverter version.